client/me/notification-settings/wpcom-settings/index.jsx

```diff
@@ -119,8 +119,8 @@ class WPCOMNotifications extends React.Component {
 				<EmailCategory
 					name={ options.news }
 					isEnabled={ get( settings, options.news ) }
-					title={ translate( 'News' ) }
-					description={ translate( 'WordPress.com news and announcements.' ) }
+					title={ translate( 'Newsletter' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'newsletter', 'block search term' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).